GWAS/GWAS_QC.ipynb

```diff
@@ -416,9 +408,11 @@
     "    kin0 <- read.table(${_output[0]:r}, header=F, stringsAsFactor=F)\n",
     "    colnames(kin0) <- c(\"FID1\",\"ID1\",\"FID2\",\"ID2\",\"NSNP\",\"HETHET\",\"IBS0\",\"KINSHIP\")\n",
```

Comment:
@haoyueshuai here in kin0 it has FID1 and FID2. you can get the ID table along the lines of:

```r
ID_table = unique(rbind(kin0[,c("FID1", "ID1")], kin0[,c("FID2", "ID2")]))
colnames(ID_table) = c("FID", "IID")
```

then match the IID to get FID